game/cards/repository.go

```diff
@@ -586,6 +588,22 @@ var DM07 = map[string]match.CardConstructor{
 	"ced03efd-90c8-4303-83b4-16d2d1198aaf": dm07.WorldTreeRootOfLife,
 }
 
+var DM08 = map[string]match.CardConstructor{
+	"701500bb-7ade-4d3d-9938-8367fa3a71bd": dm08.QuixoticHeroSwineSnout,
+	"02b4fa1f-763e-40e5-8512-bb7d2ffa322a": dm08.CorpseCharger,
+	"f9810ff9-ae5e-41c5-a88d-4bf6324e1c7e": dm08.CraniumClamp,
+	"8853aa0a-be83-4643-83dd-8be6c5d58be8": dm08.AquaRanger,
+	"7d87dcf1-468f-49cc-a5ee-dbdc533a96cc": dm08.EurekaCharger,
+	"8112be9d-50a9-4489-b3f8-257aeed62205": dm08.MagmadragonMelgars,
+	"1fbf62d7-aca7-4f0e-bf22-9170e36aad57": dm08.MuscleCharger,
+	"afca9d71-6486-4bdc-b089-159553cfb6f3": dm08.NecrodragonGalbazeek,
+	"d2f91d1a-5e8d-43ce-8512-ec6be9c3f424": dm08.NecrodragonGiland,
+	"ea9dbf9c-d049-4213-a658-e47ab25867e6": dm08.TerradragonRegarion,
+	"d031c833-da73-42a0-af4a-cd2a8ce9af6d": dm08.VolcanoCharger,
+	"cf000f38-660c-4ce0-aa24-2ecbb1e4dee2": dm08.MagmadragonJagalzor,
+	"where-does-this-come-from?":           dm08.ColiseumShell,
```

Comment:
The image ids are here: https://github.com/duel-masters/scans

Reply:
With some crazy luck the second picture I clicked happened to be what I needed. 
How should I search from that list? :D

(I'm on discord with name Huttu)